src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -55,15 +62,20 @@ public class ParameterProvider {
 
   // Lag related parameters
   public static final long MAX_CLIENT_LAG_DEFAULT = 1000; // 1 second
+  public static final long MAX_CLIENT_LAG_ICEBERG_MODE_DEFAULT = 30000; // 30 seconds
   static final long MAX_CLIENT_LAG_MS_MIN = TimeUnit.SECONDS.toMillis(1);
   static final long MAX_CLIENT_LAG_MS_MAX = TimeUnit.MINUTES.toMillis(10);
 
   public static final long MAX_ALLOWED_ROW_SIZE_IN_BYTES_DEFAULT = 64 * 1024 * 1024; // 64 MB
-  public static final int MAX_CHUNKS_IN_BLOB_AND_REGISTRATION_REQUEST_DEFAULT = 100;
+  public static final int MAX_CHUNKS_IN_BLOB_DEFAULT = 100;
+  public static final int MAX_CHUNKS_IN_REGISTRATION_REQUEST_DEFAULT = 100;
```

Comment:
is there any validation needed to throw when max_chunks_in_blob > max_chunks_in_registration_request ?

Reply:
The exception happens [here](https://github.com/snowflakedb/snowflake-ingest-java/blob/f383146763e4c7a08853b6784ba123d8f25370fb/src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java#L513) if the chunks in a blob is more than `MAX_CHUNKS_IN_REGISTRATION_REQUEST`. Didn't add a check in parameter provider since the actual number of chunks in a blob might be much smaller than the`MAX_CHUNKS_IN_BLOB`.

Reply:
Added validation during `ParameterProvdier` initialization.